apps/studio/src/lib/db/clients/sqlite.js

```diff
@@ -656,6 +662,19 @@ function getVersionString(version) {
   return version.data[0]["sqlite_version()"];
 }
 
+export async function createDatabase(conn, databaseName) {
+  if (!checkValidityOfDatabaseName(databaseName)) {
+    throw new Error('Database name invalid, must be alphanumeric / have only _ or - special characters')
+  }
+  // because this is a convenience for an otherwise ez-pz action, the location of the db file will be in the same location as the other .db files.
+  // If the desire for a "but I want this in another directory" is ever wanted, it can be included but for now this feels like it suits the current needs. 
+  const fileLocation = conn.dbConfig.database.split('/')
+  fileLocation.pop()
+
+  const db = new Database(`${fileLocation.join('/')}/${databaseName}.db`)
+  db.close()
```

Comment:
Should this switch you to using the new database file if you do this in SQLite? Maybe it should open a new window with this database open?

Reply:
Was taking the same approach we had with the others where it created it but didn't make any of the switches over. Can totally do as suggested otherwise.

Reply:
Here's what this should do for SQLite:
Back in the UI, check for SQLite, then after Create Database, submit the `AppEvent.menuClick` to the backend for 'newWindow'. You'll need to add an argument to this for the URL. Places that will be useful to look:

ClientMenuActionHandler (Example of dispatching the event)
NativeMenuActionHandler (where this action is actually handled)
WindowBuilder (see we add query params to the window URL here, this is to open the database on-open)

tl;dr - improve the handling of the `newWindow` MenuClick event to handle `OptionOptions`

For the other engines:
- it should switch to the new database